modules/tftp/server.rb

```diff
@@ -108,6 +108,28 @@ def pxeconfig_file(mac)
     end
   end
 
+  class Pxegrub2targetos < Server
+    def setup_bootloader(mac, os)
+      FileUtils.mkdir_p(pxeconfig_dir(mac))
```

Comment:
`pxeconfig_dir(mac)` is repeated a lot, store it in a separate variable?

Reply:
Done